app/src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts

```diff
@@ -40,7 +40,7 @@ export const baseItemsConflicts = {
     words: null,
     expandExtraInfo: false,
     expandedInfo1: 'This exact name is currently in use or reserved for use. A name will not be approved if there is ' +
-    'an existing business  or approved name request with the exact same name.',
+      'an existing business  or approved name request with the exact same name.',
```

Comment:
But this should be indented since it's a continuation of the previous line...?

Reply:
Yes, it looks neat a bit

Reply:
I think indenting looks better -- I find it easier to understand that indented lines go with the line above.

I recommend putting the indents back.

Reply:
Thanks. Resolving.